trade-finance-manager-api/src/v1/validation/feedback-rules/index.js

```diff
@@ -0,0 +1,25 @@
+const role = require('./role');
+const team = require('./team');
+const satisfied = require('./satisfied');
+const easyToUse = require('./easy-to-use');
+const emailAddress = require('./email-address');
+const whyUsingService = require('./why-using-service');
+
+const rules = [
+  role,
+  team,
+  whyUsingService,
+  easyToUse,
+  satisfied,
+  emailAddress,
+];
+
+module.exports = (feedback) => {
+  let errorList = {};
+
+  for (let i = 0; i < rules.length; i += 1) {
```

Comment:
this can be `for of` loop - we only have 6 items in it

Reply:
done